tests/signed-off-by-test.sh

```diff
@@ -4,10 +4,9 @@ success="true"
 
 for commit in $(git cherry master | cut -d " " -f 2)
 do
-    git show -s $commit | grep -q 'Signed-off-by:'
-    if [ $? -ne 0 ]; then
+    if ! git show -s "$commit" | grep -q 'Signed-off-by:'; then
```

Comment:
We might want to reflect these changes over in hoist's testing. Also, I think this should be a separate commit in the interest of keeping a clean tree.